cmd/crane/cmd/catalog.go

```diff
@@ -24,7 +24,7 @@ import (
 // NewCmdCatalog creates a new cobra.Command for the repos subcommand.
 func NewCmdCatalog(options *[]crane.Option) *cobra.Command {
 	return &cobra.Command{
-		Use:   "catalog",
+		Use:   "catalog {REGISTRY}",
```

Comment:
```suggestion
		Use:   "catalog [REGISTRY]",
```

Yeah let's use the same form we use elsewhere.

It might also help to add a usage section with some examples, so folks know what kinds of values to expect `[REGISTRY]` to mean.

Reply:
Updated to square brackets & included a standard example using some existing patterns.
```
crane catalog --help
List the repos in a registry

Usage:
  crane catalog [REGISTRY] [flags]

Examples:
  # list the repos for reg.example.com
  $ echo "reg.example.com" | crane catalog
  # or
  $ crane catalog reg.example.com

Flags:
  -h, --help   help for catalog

Global Flags:
      --allow-nondistributable-artifacts   Allow pushing non-distributable (foreign) layers
      --insecure                           Allow image references to be fetched without TLS
      --platform platform                  Specifies the platform in the form os/arch[/variant][:osversion] (e.g. linux/amd64). (default all)
  -v, --verbose                            Enable debug logs
```